pluginconfig/commit.go

```diff
@@ -153,6 +154,14 @@ func (c CommitOffchainConfig) Validate() error {
 		}
 	}
 
```

Comment:
I was thinking we'd call `ApplyDefaults` here in `Validate` since `Validate` is more likely to be called. And defaults are only applied on fields that need default values - wdyt?

Reply:
If we want to apply both default application and validation, we probably also need to rename the function to something like `ApplyDefaultsAndValidate`. This will avoid any surprise in the outcome.

Reply:
Yeah, that sounds good to me.